traffic_ops/app/lib/API/User.pm

```diff
@@ -281,11 +284,14 @@ sub current {
 
 	if ( &is_ldap($self) ) {
 		my $role = $self->db->resultset('Role')->search( { name => "read-only" } )->get_column('id')->single;
+		my $user_tenant_id = $self->current_user_tenant();
+		my $user_tenant = defined($user_tenant_id) ? $self->db->resultset('Tenant')->search( { id => $user_tenant_id } )->get_column('name')->single : "N/A";
```

Comment:
make sure you have a prefetch if you are going to call $row->tenant->name

so you have to make sure the tmsuer query is joined to the tenant table using a prefetch